src/hud/plugin-card/plugin-card.tsx

```diff
@@ -30,28 +31,31 @@ const Sections = styled.div`
       ${tw`border-r border-monochrome-medium-tint`}
   }`;
 
-export const PluginCard = ({ children, pluginLink, agentId }: Props) => (
-  <div tw="w-full h-fit border border-monochrome-medium-tint">
-    <div tw="flex justify-between w-full p-4 border-b border-monochrome-medium-tint text-14 leading-20">
-      <span tw="font-bold text-monochrome-default uppercase">test2code</span>
-      <Link
-        className="font-regular link no-underline"
-        to={pluginLink}
-        onClick={() => {
-          sendNavigationEvent({
-            name: NAVIGATION_EVENT_NAMES.CLICK_ON_VIEW_MORE,
-            dimension2: agentId,
-            label: "dashboards",
-          });
-        }}
-      >
-        View more &gt;
-      </Link>
+export const PluginCard = ({ children, pluginLink }: Props) => {
+  const { isAnalyticsDisabled } = useAdminConnection<AnalyticsInfo>("/api/analytics/info") || {};
```

Comment:
Create context where you wil subscribe on this topic. And inside this context you can  check isAnalyticsDisabled